framework/composer.json

```diff
@@ -54,7 +54,6 @@
     "require": {
         "php": ">=5.4.0",
         "ext-mbstring": "*",
-        "ext-mcrypt": "*",
```

Comment:
Well, if you're always getting dist versions then it's not required but if you're fetching from repos it is.